_assignments/mp4_nolab.md

```diff
@@ -0,0 +1,162 @@
+---
+layout: assignment
+title: "MP4: Flight Simulator"
+index: 41
+due: Apr. 13, 2020 @ 11:59 PM
+material:
+  -
+    name: MP4 Unity Package
+    url: /mps/MP4.unitypackage
```

Comment:
@andrewipark actually, the new unity package is >100MB. Maybe we should just put a link to the GDrive?

Link: https://drive.google.com/open?id=1jmZiOKYN9swOgJSkfBTdUmYnqHvCtN55

Reply:
fixed now